consensus/block_validation.go

```diff
@@ -86,8 +86,8 @@ func (dv *DefaultBlockValidator) ValidateSyntax(ctx context.Context, blk *types.
 	if blk.Miner.Empty() {
 		return fmt.Errorf("block %s has nil miner address", blk.Cid().String())
 	}
-	if len(blk.Ticket) == 0 {
-		return fmt.Errorf("block %s has nil ticket", blk.Cid().String())
+	if len(blk.Tickets) == 0 {
+		return fmt.Errorf("block %s has no tickets", blk.Cid().String())
```

Comment:
This should be extended to validate each ticket, so that each election proof is non-empty too.

Reply:
I think I meant VRFProof. In any case, the old code checked that the byte array was not empty, but this has been lost. Validate that the byte arrays in each ticket are not empty (better: are the expected length).